src/swamid_plugins/metainfo/collectors/oidc.py

```diff
@@ -18,6 +22,27 @@ def get_display_name(mdstore, entity_id):
     return display_name
 
 
+def get_display_names_lang(mdstore, entity_id):
+    langs = ["en", "cs"]
```

Comment:
this can be a configuration option, passed downwards to the function

```py
def get_display_names(mdstore, entity_id), langs=None:
    if not langs:
        langs = ["en"]

    # rest is the same
```

and with that in place, we can keep the "display_names" item and remove the "display_name".